bciers/apps/reporting/src/app/components/operations/OperationReviewForm.tsx

```diff
@@ -110,9 +131,28 @@ export default function OperationReviewForm({
       ),
     );
     setFormDataState(newData.report_operation);
-    setErrors(undefined);
-    setIsSnackbarOpen(true);
+
+    const reps = newData.all_representatives || [];
+    // update local available reps so UI (save button) reflects synced data
+    setAvailableReps(reps);
+    if (reps.length === 0) {
+      // set the linked helper embedded in the sentence as the single error node
+      setErrors([
+        <>
+          Before you can continue,{" "}
+          {operationRepresentativeLink(
+            formData.operation_id,
+            formData.operation_name,
+          )}{" "}
+          then return to this report
+        </>,
+      ]);
+    } else {
```

Comment:
Extra text left behind, renders twice
```suggestion
    setAvailableReps(reps);
    if (reps.length === 0) {
      // set the linked helper embedded in the sentence as the single error node
      setErrors([
          operationRepresentativeLink(
            formData.operation_id,
            formData.operation_name,
          )
      ]);
    } else {
```